src/InteractiveWindow/Editor/InteractiveWindow.cs

```diff
@@ -1561,25 +1527,16 @@ private ReplSpanKind GetSpanKind(ITextSnapshot snapshot)
             {
                 return ReplSpanKind.StandardInput;
             }
-            if (textBuffer == _promptBuffer)
-            {
-                return ReplSpanKind.Prompt;
-            }
-            if (textBuffer == _secondaryPromptBuffer)
+            if (textBuffer.ContentType == _inertType)
             {
-                return ReplSpanKind.SecondaryPrompt;
-            }
-            if (textBuffer == _standardInputPromptBuffer)
-            {
-                return ReplSpanKind.StandardInputPrompt;
+                return ReplSpanKind.None;
             }
             return ReplSpanKind.Language;
         }
 
         private bool IsPrompt(ITextSnapshot snapshot)
         {
-            var kind = GetSpanKind(snapshot);
-            return (kind == ReplSpanKind.Prompt) || (kind == ReplSpanKind.SecondaryPrompt) || (kind == ReplSpanKind.StandardInputPrompt);
+            return snapshot.ContentType == _inertType;
```

Comment:
Doesn't ```_inertType``` indicate a read-only buffer? (assuming it does, "_inertType" or "_readOnlyBuffer" type seems most appropriate to me)